src/app/donation-start/donation-start-form/donation-start-form-parent.component.ts

```diff
@@ -1858,4 +1862,13 @@ export class DonationStartFormParentComponent implements AfterContentChecked, Af
     this.prefillRarelyChangingFormValuesFromPerson(person);
     this.loadFirstSavedStripeCardIfAny(id, jwt);
   }
+
+  onDonationSliderMove = (tipPercentage: number, tipAmount: number) => {
+    this.tipPercentage = tipPercentage;
```

Comment:
maybe something to note on Jira and fix later rather than here - but I'm slightly concerned that taking the percentage means we don't have the rounding that's done for display inside the slider, so the amount they actually pay could be a few pence different to what they expected. We should round once and then use that output for display and for actually taking the money.

I think maybe we should get rid of the tipPercentage field in the class and make everything work off the tipAmount.